CHANGELOG.md

```diff
@@ -10,7 +10,11 @@
 
 #### Enhancements
 
-* None.
+* Add `nsobject_prefer_isequal` rule to warn against implementing `==` on an
+  `NSObject` subclass as calling `isEqual` (i.e. when using the class from
+  Objective-C) will will not use the defined `==` method.
```

Comment:
requires two trailing spaces as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes